### PR TITLE
common: proto dep update

### PR DESCRIPTION
**Pull request by lambdai**
Find build error while I am working on github.com/istio/tools/pull/1565

The goal of this PR is fix known errors with github.com/istio/tools/pull/1565 as a proof of success of building kafka network filters.

We may need more PRs to update the proxy code base to accommodate envoyproxy.
  
Signed-off-by: Yuchen Dai lambdai@google.com

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -39,9 +39,9 @@ bind(
 #
 # Note: this is needed by release builder to resolve envoy dep sha to tag.
 # Commit date: 2021-05-13
```
Comment (bianpengyuan) [inline WORKSPACE:41]: nit: update the date.

Comment (lambdai) [reply WORKSPACE]: Nice catch! Update.

**Comment by lambdai** [comment]
so far so good

```
[9,808 / 12,916] Compiling external/envoy/source/extensions/filters/network/kafka/external/kafka_response_resolver.cc; 35s remote-cache, linux-sandbox ... (126 actions, 75 running)
```

**Pull request closed without merging**